include/aspect/simulator/assemblers/stokes.h

```diff
@@ -153,6 +153,28 @@ namespace aspect
                 internal::Assembly::CopyData::CopyDataBase<dim> &data) const;
     };
 
+
+    /**
+     * This class assembles the right-hand-side term of the Stokes equation
+     * that is caused by the compressibility in the mass conservation equation.
```

Comment:
mention that it includes both temperature and pressure? In the old pull request, we had 

> caused by the compression based on the changes in pressure and temperature in the mass conservation equation